component/loki/source/heroku/heroku.go

```diff
@@ -67,26 +69,30 @@ type Component struct {
 	opts    component.Options
 	metrics *ht.Metrics
 
-	mut    sync.RWMutex
-	args   Arguments
-	fanout []loki.LogsReceiver
-	target *ht.HerokuTarget
+	mut           sync.RWMutex
+	args          Arguments
+	fanout        []loki.LogsReceiver
+	target        *ht.HerokuTarget
+	serverMetrics *util.UncheckedCollector
```

Comment:
Nit: `serverMetrics` technically doesn't require `mut` for locking since it has its own lock, so we can move it up in the declarations list next to `metrics` and other fields.

Reply:
Totally agree. This is something I wish we had a linter for :)